requirements/extras/test_requirements.txt

```diff
@@ -12,7 +12,7 @@ awslogs==0.14.0
 black==22.3.0
 stopit==1.1.2
 # Update tox.ini to have correct version of airflow constraints file
-apache-airflow==2.6.0
+apache-airflow==2.6.3
```

Comment:
As per comment above can you also update the tox.ini file for this apache-airflow version?

Reply:
Ok sounds good let me take a look